gnocchi/rest/__init__.py

```diff
@@ -665,6 +665,9 @@ def post(self):
         except indexer.NoSuchResource as e:
             abort(404, e)
 
+        return pecan.request.indexer.get_resource(
```

Comment:
If the user does not need this output it's a whole request for nothing.
It might  be better to leave the user request it if it needs it after, no?